addons/point_of_sale/static/src/app/store/models.js

```diff
@@ -2463,9 +2360,6 @@ export class Order extends PosModel {
     set_screen_data(value) {
         this.screen_data["value"] = value;
     }
-    get_current_screen_data() {
-        return this.screen_data["value"] ?? { name: "ProductScreen" };
-    }
```

Comment:
I think this is used in the customer display template

Reply:
good catch! I added it back